server.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+
+const cowsay = require('cowsay');
+const parseBody = require('./lib/parse-body.js');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer(function(req,res){
+  req.url = url.parse(req.url);
+  req.url.query = queryString.parse(req.url.query);
+  console.log('req.url', req.url);
```

Comment:
@loganlsr best practice is to remove console.log's from your master branch prior to pushing and/or deploying - I usually add these back in, on a contained branch, that is used simply for testing (ie: features/testing-requests)